CONTRIBUTING.md

```diff
@@ -162,162 +268,115 @@ fixes:
 
+```python
+  @deprecated_function(last_version="0.3", msg="Use new_function instead.")
+  def old_function(*args, **kwargs):
+      pass
+  def new_function(*args, **kwargs):
+      pass
 ```
-tox -epy37 -- -n test/python/test_examples.py
-```
-to run a class:
 
-```
-tox -epy37 -- -n test.python.test_examples.TestPythonExamples
-```
-to run a method:
-```
-tox -epy37 -- -n test.python.test_examples.TestPythonExamples.test_all_examples
-```
-
-##### STDOUT/STDERR and logging capture
-
-When running tests in parallel using `stestr` either via tox, the Makefile
-(`make test_ci`), or in CI we set the env variable
-`QISKIT_TEST_CAPTURE_STREAMS` which will capture any text written to stdout,
-stderr, and log messages and add them as attachments to the tests run so
-output can be associated with the test case it originated from. However, if
-you run tests with `stestr` outside of these mechanisms by default the streams
-are not captured. To enable stream capture just set the
-`QISKIT_TEST_CAPTURE_STREAMS` env variable to `1`. If this environment
-variable is set outside of running with `stestr` the streams (STDOUT, STDERR,
-and logging) will still be captured but **not** displayed in the test runners
-output. If you are using the stdlib unittest runner a similar result can be
-accomplished by using the
-[`--buffer`](https://docs.python.org/3/library/unittest.html#command-line-options)
-option (e.g. `python -m unittest discover --buffer ./test/python`).
+To deprecate a class:
 
-### Code style
-
-The qiskit-experiments repo uses black for code formatting and style and pylint
-is run to do a deeper check on the code. You can run these checks locally with
-
-```
-tox -elint
+```python
+  @deprecated_class(last_version="0.3", new_cls=NewCls)
+  class OldClass:
+      pass
+  class NewClass:
+      pass
 ```
 
-If there is a code formatting issue identified by black you can just run
-``black`` locally to fix this (or ``tox -eblack`` which will install it and
-run it).
-
-Because `pylint` analysis can be slow, there is also a `tox -elint-incr` target,
-which only applies `pylint` to files which have changed from the source github.
-On rare occasions this will miss some issues that would have been caught by
-checking the complete source tree, but makes up for this by being much faster
-(and those rare oversights will still be caught by the CI after you open a pull
-request).
+This will inform the user which version of Qiskit Experiments will remove the 
+deprecated class or function.
 
-### Development Cycle
+### Development cycle
 
-The development cycle for qiskit-experiments is all handled in the open using
-the project boards in Github for project management. We use milestones
-in Github to track work for specific releases. The features or other changes
+The development cycle for Qiskit Experiments is all handled in the open using project boards in Github for project management. We use milestones
```

Comment:
Maybe add a link to the milestone page? I know users less familiar with github often struggle to find this

Reply:
Good point, thanks!